docker.nix

```diff
@@ -287,7 +288,7 @@ pkgs.dockerTools.buildLayeredImageWithNixDb {
 
   inherit name tag maxLayers uid gid uname gname;
 
-  contents = [ baseSystem ];
+  contents = [ baseSystem ] ++ extraBasePkgs;;
```

Comment:
iirc passing more than one element turns the whole root file system into a symlink farm.
I doubt that this is correct or intended.

Reply:
Could you elaborate on what this will change will cause? In my understanding, this is the intended way of how nix docker image building works.

Reply:
Only partly. We also need things in the root file system that aren't symlinks, such as `/tmp` and `/var` which are created as part of `baseSystem`.

Reply:
Yes, I understand. So how are additional packages interfering with this? Isn't this what's happening on NixOS every time I install a package?

Reply:
NixOS doesn't have a `contents` that fills the root file system. It sets up various parts of the root file system on installation, on boot, and on activation.
[dockerTools](https://nixos.org/manual/nixpkgs/stable/index.html#ssec-pkgs-dockerTools-buildLayeredImage) is fairly low level and doesn't do many of the things NixOS does.